docs/Users_Guide/appendixH.rst

```diff
@@ -0,0 +1,87 @@
+Unstructured Grid Files
+=======================
+
+Support for UGRID files in NetCDF format is provided. MET will attempt to auto-detect UGRID files. If the file is CF-compliant and the “Conventions” global attribute is “UGRID” or “MPAS”, or if the global attribute “mesh_spec” is present, MET recognizes the file as a UGRID file. If the user is not using a supported UGRID, or a UGRID file that does not contain the required global attributes for autodetection, the user should set the **file_type** configuration entry to **NETCDF_UGRID**. In some cases, the UGRID topology is provided in an ancillary file separate from forecast variables. This is controlled via the **ugrid_coordinates_file** configuration entry. This file is checked first, and then the UGRID data file. If the same elements are found in both files, the information from the data file takes precedence, and thus it is recommended to leave **ugrid_coordinates_file** unset if all required elements are found in the data file. If both a data and coordinates file are provided, the shape of the latitude and longitude dimensions are cross-referenced as a limited enforcement measure that the coordinates file and data file belong to the same UGRID.
+
+Required Unstructured Grid Metadata
+===================================
+
+To correctly parse the UGRID topology, MET needs the following information that must be contained within the UGRID coordinates file or the UGRID data file. It is often the case that the variable names for these elements differ between UGRIDs. If the user is not using one of the supported UGRIDs, they will need to define the **ugrid_metadata_map** in a custom configuration file that they provide on the command line using the **-ugrid_config** command line argument, which tells MET the variable names the correspond to the following elements:
+
+.. _table_ugrid_metadata:
+
+.. list-table:: Required UGRID Metadata
+  :widths: auto
+  :header-rows: 1
+
+  * - Metadata Map Key
+    - Description
+    - Required
+  * - dim_face
+    - Dimension name for UGRID cells
+    - Required
+  * - lat_face
+    - Coordinate variable name for the latitude of each UGRID cell
+    - Required
+  * - lon_face
+    - Coordinate variable name for the longitude of each UGRID cell
+    - Required
+  * - dim_time
+    - Dimension name for the time coordinate
+    - Required
+  * - time
+    - Coordinate variable for time
+    - Required
+  * - dim_vert
+    - Dimension name for the vertical coordinate, if present
+    - Optional
+  * - vert_face
+    - Coordinate variable name for the vertical coordinate
+    - Optional
+
+Unstructured Grid Configuration Entries
+=======================================
+
+In the PointStat and GridStat config files, a user can control aspects of the UGRID capability. Since the UGRID support in MET is optional, these items are not included in the default MET config files. If the user requires modification for any of these, they must add them to their MET config file.
```

Comment:
Suggest "If the user requires modification for any of **these**" --> "If the user requires modification for any of **the following**"